rspec-mode.el

```diff
@@ -505,7 +505,7 @@ to navigate to the example or method corresponding to point."
   "Return A-FILE-NAME but converted in to a spec file name."
   (concat
    (file-name-directory a-file-name)
-   (replace-regexp-in-string "\\(\\.rb\\)?$" "_spec.rb" (file-name-nondirectory a-file-name))))
+   (replace-regexp-in-string "\\(\\.\\(rb\\|rake\\)\\)?$" "_spec.rb" (file-name-nondirectory a-file-name))))
```

Comment:
The recent commit I added *should* do that, but it's still breaking on my machine. Hope to solve it until tomorrow. In the meantime, if you can find a solution, it'd be great.